src/dsp_tools/excel2xml.py

```diff
@@ -1217,7 +1217,20 @@ def make_text_prop(
             **kwargs,
             nsmap=xml_namespace_map
         )
-        value_.text = val.value
+        if kwargs["encoding"] == "utf8":
+            # write the text into the tag, without validation
+            value_.text = str(val.value)
+        else:
+            # enforce that the text is well-formed XML: serialize tag ...
+            content = etree.tostring(value_, encoding="unicode")
+            # ... insert text at the very end of the string, and add an ending tag to the previously single <text/> tag ...
+            content = re.sub(r"/>$", f">{val.value}</text>", content)
+            # ... try to parse it again
+            try:
+                value_ = etree.fromstring(content)
```

Comment:
it looks much neater now (and I can of course live with that). But it still acts in a complicated way:
1. create an etree Element
2. serialize that element to a string
3. add the provided value to the string by means of replacing
4. parse the modified string to an etree Element.

IMO it should be:
1. create an Element
2. parse the provided content to an etree Element structure
3. add this structure as children to the Element

If that proves tricky, then go ahead and merge it like this... but I think the other way would be cleaner.